TypeCobol.Test/TestUtils.cs

```diff
@@ -40,5 +46,89 @@ public static void compareLines(string testName, string result, string expectedR
                 throw new Exception(errors.ToString());
             }
         }
+
+        public static string GetReportDirectoryPath()
+        {
+            string pwd = Directory.GetCurrentDirectory();
+            return Path.Combine(Directory.GetParent(pwd)?.FullName, _report);
```

Comment:
As the report will change depending on the build configuration (Debug or Release), can you move the report under the folder of the build configuration:
- Bin\Debug\PerformanceReports
- Bin\Release\PerformanceReports